### ISSUE TITLE
Onboarding stage of Fenix migration

### ISSUE THREAD

**Issue by liuche** [mozilla-mobile/fenix#7093]
### Why/User Benefit/User Problem
As a user migrated from Firefox, I want to have some onboarding explanation for what this "new" browser is. (And if I am a brand new Firefox user, I want to have a good onboarding experience as well that doesn't contain irrelevant information)

### Dependencies
- [ ] Do we want a search engine card?
- [ ] What circumstances do we show these cards? (in addition to migrated vs non-migrated)
- [ ] Final copy
Max 3 migration-only onboarding cards 

### Acceptance Criteria (how do I know when I’m done?)
- [ ] Add new cards and link to setting (if applicable)
- [ ] Update settings (if applicable) based on cards
- [ ] Telemetry for new cards (if not built in to card template)

**Onboarding** section of https://miro.com/app/board/o9J_kw1CGow=/

**Comment by liuche** [comment]
Waiting on decision of whether we should have a search onboarding card.

**Comment by mheubusch** [comment]
@liuche No search onboarding card at this time.  One new See what's new card here: https://docs.google.com/document/d/1ynhu3K-u4unlHBcf0bjW7PHJcoBBa1UV1-PZfHH1LjM/edit?usp=sharing

<img width="198" alt="Screen Shot 2020-01-15 at 1 55 54 PM" src="https://user-images.githubusercontent.com/20115914/72466577-ca807c80-379e-11ea-9278-620a6078b6ad.png">

**Comment by vesta0** [comment]
Closing this issue since we will not have any migration specific onboarding cards.

**Comment by liuche** [comment]
Need to pre-land these strings.

**Comment by liuche** [comment]
I just talked to @mheubusch and it looks like there are going to be a few tweaks to these cards. Strings for the 3 new cards are [here](https://docs.google.com/document/d/1ynhu3K-u4unlHBcf0bjW7PHJcoBBa1UV1-PZfHH1LjM/edit#) (scroll down).

@vesta0 I think we should still get these strings pre-landed ASAP, assuming even though they aren't specifically for migration, they are new onboarding cards that we should have localized for Beta Migration.

**Comment by vesta0** [comment]
I am closing this issue because it is a dup of #6980
